### ISSUE TITLE
Error when use in Symfony project

### ISSUE THREAD

**Issue by fd6130** [tzsk/otp#132]
I got namespace error when i try to use it in my Symfony project.

Namespace error as below:

```php
Attempted to load class "Repository" from namespace "Illuminate\Cache".
Did you forget a "use" statement for e.g. "Google_Service_ArtifactRegistry_Repository", "Illuminate\Contracts\Config\Repository" or "Illuminate\Contracts\Cache\Repository"?
```

My code:

```php
class TzskOtpManager
{
    private $generator;

    public function __construct()
    {
        $this->generator = otp('../otp-tmp');
    }
}
```

Do i need to include any namespace when using `otp('../otp-tmp');` in my code?

**Comment by fd6130** [comment]
Seems like we have to manually install two things if we use it non-laravel project:

```
composer require illuminate/cache
composer require illuminate/filesystem
``` 

It fix the namespace error but i'm not sure whether it is correct or not. Need feedback about this.

**Comment by tzsk** [comment]
That's really weird. Let me investigate. In the meantime you'll be fine installing `illuminate/support` manually.

**Comment by fd6130** [comment]
> That's really weird. Let me investigate. In the meantime you'll be fine installing `illuminate/support` manually.

This is the result in my terminal when i `composer require tzsk/otp`, and after that namespace error occurs when i use it.

![image](https://user-images.githubusercontent.com/11419744/108468609-f5060880-72c1-11eb-9023-31fe35c5b839.png)

**Comment by tzsk** [comment]
Ideally illuminate/contact should pull those in. Anyway would you mind adding cache and filesystem in the composer.json and creating a PR?

**Comment by fd6130** [comment]
> Ideally illuminate/contact should pull those in. Anyway would you mind adding cache and filesystem in the composer.json and creating a PR?

Sure, will create a PR shortly.

**Comment by fd6130** [comment]
What version constraint shall i put for cache and filesystem in the composer.json?

**Comment by tzsk** [comment]
Yup. Perfect.

**Comment by tzsk** [comment]
Released `v5.1.1`
